php-custom-rules/build.gradle.kts

```diff
@@ -0,0 +1,56 @@
+import org.sonarsource.php.registerCleanupTask
+
+plugins {
+  id("java-library")
+  alias(libs.plugins.shadow)
+}
+
+dependencies {
+  compileOnly(libs.sonar.plugin.api)
+  compileOnly(project(":sonar-php-plugin"))
```

Comment:
There should not be a dependency to `sonar-php-plugin` but rather to `php-frontend`.